### ISSUE TITLE
TMC2209 and Diag-Pins with Endstops

### ISSUE THREAD

**Issue by Bastlwastl84** [FYSETC/FYSETC-SPIDER#7]
Does the Diag-Pins from the Stepper needed to bend by side or cutted of if using the Fystec TMC2209 V3.1 with the Spider V1 WITH Endstops?

**Comment by GerogeFu** [comment]
No, you only need to close the diag pin jumper if you want to use sensorless homing feature. Or remove the jumper if you want to use endstop.
![image](https://user-images.githubusercontent.com/7496375/120442448-34132200-c3b8-11eb-818a-5993b6f22269.png)
